SourceryFramework/Sources/Parsing/Utils/InlineParser.swift

```diff
@@ -28,33 +30,36 @@ public enum TemplateAnnotationsParser {
         return (bridged as String, annotatedRanges)
     }
 
-    public static func annotationRanges(_ annotation: String, contents: String, aggregate: Bool = false) -> (annotatedRanges: [String: [NSRange]], rangesToReplace: Set<NSRange>) {
+    public static func annotationRanges(_ annotation: String, contents: String, aggregate: Bool = false) -> (annotatedRanges: AnnotatedRanges, rangesToReplace: Set<NSRange>) {
         let bridged = contents.bridge()
         let regex = try? self.regex(annotation: annotation)
 
         var rangesToReplace = Set<NSRange>()
-        var annotatedRanges = [String: [NSRange]]()
+        var annotatedRanges = AnnotatedRanges()
 
         regex?.enumerateMatches(in: contents, options: [], range: bridged.entireRange) { result, _, _ in
-            guard let result = result, result.numberOfRanges == 5 else {
+            guard let result = result, result.numberOfRanges == 6 else {
                 return
             }
 
-            let nameRange = result.range(at: 2)
-            let startLineRange = result.range(at: 3)
-            let endLineRange = result.range(at: 4)
+            let indentationRange = result.range(at: 2)
+            let nameRange = result.range(at: 3)
+            let startLineRange = result.range(at: 4)
+            let endLineRange = result.range(at: 5)
 
+            let indentation = bridged.substring(with: indentationRange)
+            print("indentation raw \(indentation) \(indentation.count)")
```

Comment:
can we remove this debug print statement?

Reply:
Of course! Oops... 😅